src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -17,6 +17,10 @@
 //  SVE Intrinsics
 
 // Sve
+HARDWARE_INTRINSIC(Sve,           Abs,                                                              -1,     -1,      false, {INS_sve_abs,        INS_invalid,        INS_sve_abs,        INS_invalid,        INS_sve_abs,        INS_invalid,        INS_sve_abs,        INS_invalid,        INS_sve_fabs,       INS_sve_fabs},    HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation)
+
+HARDWARE_INTRINSIC(Sve,           Add,                                                              -1,     -1,      false, {INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_fadd,       INS_sve_fadd},    HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation|HW_Flag_HasRMWSemantics|HW_Flag_LowMaskedOperation)
```

Comment:
Why does this one in particular need `EmbeddedMaskedOperation`, the manual has entries for both `Add (vectors, predicated)` and `Add (vectors, unpredicated)`. Both entries look to be for SVE1